docs/guide/introduction.md

```diff
@@ -16,7 +16,84 @@ In short, it recognizes pain points on both sides of the CMS experience and reli
 
 ## The module system
 
-Vestibulum id ligula porta felis euismod semper. Maecenas faucibus mollis interdum. Cras mattis consectetur purus sit amet fermentum. Fusce dapibus, tellus ac cursus commodo, tortor mauris condimentum nibh, ut fermentum massa justo sit amet risus. Donec ullamcorper nulla non metus auctor fringilla. Cras mattis consectetur purus sit amet fermentum. Donec ullamcorper nulla non metus auctor fringilla.
+As you learn to use ApostropheCMS, you'll quickly see how all server-side code is organized in a system of "modules." Modules are the building blocks we use to build Apostrophe projects — as well as the CMS itself.
```

Comment:
```suggestion
In Apostrophe, all server-side code is organized in a system of "modules." Modules are the building blocks we use to build Apostrophe projects — as well as the CMS itself.
```
Removed the first bit because it's not necessary to understanding, imo. I also wonder if there is a more direct way to say 'server-side code'....is it implicit? Idk. 

```suggestion
Apostrophe is organized in a system of modules. Modules are the building blocks we use to build Apostrophe projects — as well as the CMS itself.
```